### DISCUSSION TITLE
Logic App standard and Service Bus Connector protocol

### DISCUSSION THREAD

**Discussion by xaviergxf** [Azure/logicapps#540] [Q&A]
Hi,

  I couldn't find anywhere on the documentation what is the default protocol used on Service Bus Connector while using Logic App Standard plan. Do you have a documentation about that?

Thanks,
Best regards

**Comment by divyaswarnkar** [comment]
Hi @xaviergxf , the service bus connector is for Azure Service Bus and therefore supports the protocols supported by Azure Service Bus - which is AMQP 1.0

**Reply by apranaseth** [reply]
The default is Amqp you can change to AmqpWebSockets if needed using the transporttype in the host.json. Refer [this ](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-service-bus?tabs=in-process%2Cextensionv5%2Cextensionv3&pivots=programming-language-csharp#hostjson-settings) for more details.

**Reply by xaviergxf** [reply]
Thanks! When searching for the answer on the docs i couldn't find anything related to "logic app". Shouldn't this be on the logic app docs as well?
